go.mod

```diff
@@ -2,6 +2,8 @@ module github.com/crossplane/crossplane
 
 go 1.13
 
+replace github.com/crossplane/crossplane-runtime => github.com/muvaf/crossplane-runtime v0.0.0-20200429141134-1ddc2c5d8d89
```

Comment:
Mental note: remove this before merge. I believe the changes are in crossplane-runtime now.